samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js

```diff
@@ -140,8 +111,30 @@
                         .then((result) => {
                             // Consent succeeded - use the token we got back
```

Comment:
Dont use this token. Instead get back and retry

Reply:
made changes to show refresh button.